htdocs/main.inc.php

```diff
@@ -1039,6 +1039,16 @@ function analyseVarsForSqlAndScriptsInjection(&$var, $type)
 		$user->rights->user->user->supprimer = 1;
 		$user->rights->user->self->creer = 1;
 		$user->rights->user->self->password = 1;
+
+		//Required if advanced permissions are used with MAIN_USE_ADVANCED_PERMS
```

Comment:
So a test 
if (!empty($conf->global->MAIN_USE_ADVANCED_PERMS)) {
may be welcome here.

Reply:
much better, thank you